src/core/events/bpf/include/events.h

```diff
@@ -76,6 +77,19 @@ static __always_inline void *get_event_section(struct trace_raw_event *event,
 	return section;
 }
 
+/* Similar to get_event_section but initialize the section data to 0s. */
+static __always_inline void *get_event_zsection(struct trace_raw_event *event,
+						u8 owner, u8 data_type, u16 size)
+{
+	void *section = get_event_section(event, owner, data_type, size);
+
+	if (!section)
+		return NULL;
+
+	memset(section, 0, size);
```

Comment:
This raises a warning in clang, we could either  macro it or use  `__builtin_memset` or even `__builtin_memset_inline`